src/asset_manager/view/AssetImageView.js

```diff
@@ -79,7 +79,7 @@ module.exports = require('./AssetView').extend({
    * @private
    * */
   onRemove(e) {
-    e.stopPropagation();
+    e.stopImmediatePropagation();
```

Comment:
I thought that e.stopPropagation() would work, but found this after a quick google search: https://stackoverflow.com/questions/5457915/how-can-i-stop-event-propagation-with-backbone-js

Reply:
Great, this is actually working 👍